scripts/templates/nxiq-operator-certified.vX.X.X-X.clusterserviceversion.yaml

```diff
@@ -141,6 +141,8 @@ spec:
     Once the server instance is created by the operator and running,
     you'll want to expose the service as you see fit:
     1. Create a Route to the new service for iq.applicationPort (8070).
+    2. Visit the URL provided by the Route, and login with the default credentials:
+      `admin`/`admin123`
```

Comment:
Are the backticks formatting? Wondering if this ends up being markdown or what and mainly if the README should have backticks as well. No biggie though.

Reply:
good point on readme. fixed it. if you expand this block to see following lines, you can see the table full of back-tick formatting that's valid in this description.